Userland/Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.cpp

```diff
@@ -2460,6 +2460,8 @@ void CalculatedStyleValue::CalculationResult::multiply_by(CalculationResult cons
 {
     // We know from validation when resolving the type, that at least one side must be a <number> or <integer>.
     // Both of these are represented as a double.
+    //FIXME:context should not be empty
+    if(!context.has_value()) return;
```

Comment:
As Matt said, I'm not sure if this is the correct way to fix this. However, if it is: this code should go in the `[&](Length const& length) {` lambda below, not here. We only use a `Length::ResolutionContext` when dealing with lengths. In other cases, it will always be empty and that's fine.